worksheet_D/oxo.py

```diff
@@ -0,0 +1,222 @@
+class OxoBoard:
+    def __init__(self):
+        """
+        For nxn I would have to choose a different
+        naming scheme something like ZeroZero,OneZero representing x and y
+        """
+
+            # I use the dictionary so I can have names next to my values for ease of changing them
+        self.fields = {"Top Left": 0, "Top Middle": 0, "Top Right": 0,
+                       "Middle Left": 0, "Middle Middle": 0, "Middle Right": 0,
+                       "Bottom Left": 0, "Bottom Middle": 0, "Bottom Right": 0}
+
+    def get_square(self, x, y):
+        """
+        For nxn I would just have to add the extra squares
+        """
+            # Here I'm returning the current value of the current position
+
+            # Top line
+        if x == 0 and y == 0:
```

Comment:
If you had used tuples as dictionary keys, you could replace this long `if` statement with a single line:
```
return self.fields[x,y]
```